client/app/scripts/liveblog-advertising/views/main.html

```diff
@@ -53,46 +54,134 @@ <h6 class="pull-left">
 	</div>
 </section>
 
+<!-- MODAL FOR ADDING EDITING ADVERTS -->
 <div sd-modal data-model="advertModalActive">
-        <div class="modal-header">
-            <button class="close" ng-click="cancelAdvertCreate()"><i class="icon-close-small"></i></button>
-            <h3 translate>{{ advertType.name }}</h3>
+    <div class="modal-header">
+        <button class="close" ng-click="cancelAdvertCreate()"><i class="icon-close-small"></i></button>
+        <h3 translate>{{ advertType.name }}</h3>
+    </div>
+    <div class="modal-body" style="min-height: 300px;">
+        <div class="item-creation">
+            <fieldset>
+                <div class="field">
+                    <label translate>Advert Name<span class="ft-alert" translate ng-if="!advert.name">required</span></label>
+                    <input type="text" ng-model="advert.name" maxlength="24" required>
+                </div>
+                <div class="field">
+                    <freetype-render
+                    freetype="advertType"
+                    freetype-data="freetypesData"
+                    validation="validation"
+                    control="freetypeControl"></freetype-render>
+                </div>
+            </fieldset>
         </div>
-        <div class="modal-body" style="min-height: 300px;">
-            <div class="item-creation">
-                <fieldset>
-                    <div class="field">
-                        <label translate>Advert Name<span class="ft-alert" translate ng-if="!advert.name">required</span></label>
-                        <input type="text" ng-model="advert.name" maxlength="24" required>
-                    </div>
-                    <div class="field">
-                        <freetype-render
-                        freetype="advertType"
-                        freetype-data="freetypesData"
-                        validation="validation"
-                        control="freetypeControl"></freetype-render>
-                    </div>
-                </fieldset>
-            </div>
+    </div>
+    <div class="modal-footer">
+        <div>
+            <button type="button" class="btn btn-default pull-left" translate ng-click="cancelAdvertCreate()">Cancel</button>
+            <button type="button" class="btn btn-primary btn-info" translate ng-click="saveAdvert()"
+            ng-disabled="!advert.name || dialogAdvertLoading"
+            >Save</button>
         </div>
-        <div class="modal-footer">
-            <div>
-                <button type="button" class="btn btn-default pull-left" translate ng-click="cancelAdvertCreate()">Cancel</button>
-                <button type="button" class="btn btn-primary btn-info" translate ng-click="saveAdvert()"
-                ng-disabled="!advert.name || dialogAdvertLoading"
-                >Save</button>
-            </div>
+    </div>
+</div>
+
+<!-- MODAL FOR ADDING EDITIONG COLLECTIONS -->
+<div sd-modal data-model="collectionModalActive" class="advertising">
```

Comment:
same as above " It can be done as another improvement task to move all modals from main views as most modules use this approach. "